src/cpr_data_access/models.py

```diff
@@ -196,14 +196,22 @@ def to_string(self) -> str:
         """Return text in a clean format"""
         return " ".join([line.strip() for line in self.text])
 
+    def __hash__(self) -> int:
+        """Get hash of the text-block. Based on the text and the text_block_id"""
+        text_utf8 = self.to_string().encode("utf-8")
+        hash_string = (
+            hashlib.md5(text_utf8).hexdigest()
```

Comment:
Out of interest why not the built in `hash` function?

Reply:
Good point -- I've started off by thinking of merging `__hash__` and `text_hash` but since we're separating out the usecases, there's no good reason not to use the built-in function -- updated!